requirements.txt

```diff
@@ -0,0 +1,7 @@
+BeautifulSoup==3.2.1
+dj-database-url==0.4.1
```

Comment:
not totally sure, it helps in case of databases, related to environment variables that will be used by heroku .